composure.sh

```diff
@@ -329,9 +329,10 @@ glossary ()
   group 'composure'
 
   typeset targetgroup=${1:-}
-  typeset maxwidth=$(_longest_function_name_length | awk '{print $1 + 5}')
+  typeset functionlist="$(_typeset_functions)"
+  typeset maxwidth=$(_longest_function_name_length "$functionlist" | awk '{print $1 + 5}')
 
-  for func in $(_typeset_functions); do
+  for func in $functionlist; do
```

Comment:
Zsh treats this like a single string, and does not iterate over the `functionlist`. The code below works:
```
for func in $(_typeset_functions_about); do
```